plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerConfig.java

```diff
@@ -52,4 +53,19 @@ public FakerConfig setDefaultLimit(long value)
         this.defaultLimit = value;
         return this;
     }
+
+    @Min(1)
+    @Max(1_000)
+    public int getMinSplits()
+    {
+        return minSplits;
+    }
+
+    @Config("faker.min-splits")
```

Comment:
Why limit number of rows per split? Also, using the number of cpus from the coordinator seems wrong.

Reply:
> Why limit number of rows per split?

For better parallelism, i.e., use more splits to generate the same total row count.

> using the number of cpus from the coordinator seems wrong

We use that because worker CPU count is harder to fetch and typical deployment uses same or smaller size coordinator than workers. It's a better default than some arbitrary number.

Reply:
I get it now, thanks! But I still think the config name should be for the minimum number of splits per node - if we add an artificial number of rows per split, then we can generate more splits.

I also set the default to 4. I'd rather have it constant, then depend on the coordinator, not worker node's CPU count. It should not matter for low number of rows, and for high numbers the `MAX_ROWS_PER_SPLIT` would make sure we have enough splits generated.